index.html

```diff
@@ -227,8 +215,29 @@ <h3 class="qualification-heading"><i class="fa-solid fa-circle"></i>Skills</h3>
           </ul>
       </div>
     </section>
+    <section class="contact">
+      <h3 class="contact-heading">
+        I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.
+      </h3>
+    <form action="https://formspree.io/f/xgebvavj" method="POST" class="contact-form">
+      <input type="text" class="contact-input" placeholder="Full name" maxlength="30" required>
+      <input type="email" class="contact-input" placeholder="Email address" required>
+      <textarea name="textarea" class="contact-textarea" id="" cols="30" rows="10" required maxlength="500" placeholder="Enter text here">Enter text here</textarea>
```

Comment:
- [ ] As required by the previous reviewer, kindly remove the text I underlined in the image below:

![image](https://user-images.githubusercontent.com/86200356/214574753-e4624f08-8d45-4d7f-a576-f6b427cbd041.png)